contracts/GenArt721CoreV3.sol

```diff
@@ -1049,15 +1087,15 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
         )
     {
         // calculate revenues
-        artblocksRevenue_ = (_price * artblocksPercentage) / 100;
+        artblocksRevenue_ = (_price * artblocksPrimarySalesPercentage) / 100;
         uint256 projectFunds = _price - artblocksRevenue_;
         additionalPayeePrimaryRevenue_ =
             (projectFunds *
                 projectIdToAdditionalPayeePrimarySalesPercentage[_projectId]) /
             100;
         artistRevenue_ = projectFunds - additionalPayeePrimaryRevenue_;
         // set addresses from storage
-        artblocksAddress_ = artblocksAddress;
+        artblocksAddress_ = artblocksPrimarySalesAddress;
```

Comment:
@jakerockland - do you prefer that I update this return var name to be `artblocksAddress` -> `artblocksPrimarySalesAddress`? would go into the minter contracts as well. I'm fine either way; since this function is getting primary sales splits, designating something as primary is a bit redundant, but we are doing it that way for additional payee, so could also do it for Art Blocks.

Reply:
In isolation, I don't think I have a strong opinion on this one–though in alignment my comment here https://github.com/ArtBlocks/artblocks-contracts/pull/237#issuecomment-1213652892 I think we may want to maintain backwards compatibility for these royalty reading functions, in which case perhaps slight preference to not changing the naming convention here.

Reply:
@jakerockland 

Good point!. After not implementing this, then implementing this, I have a minor preference to keep the new naming to improve readability of V3 (mostly because everything for AB & artist additional payee is labeled as either primary or secondary).

re [#237 comment](https://github.com/ArtBlocks/artblocks-contracts/pull/237#issuecomment-1213652892), if we opt to keep the legacy getRoyaltiesData function, WDYT about adding a backwards-compatible getter function for `artblocksAddress` that returns `artblocksPrimarySalesAddress`, and document the getter function accordingly?

Reply:
per discussion, will add a getter for `artblocksAddress`, then resolve this 😄